test/puppetlabs/puppetdb/integration/storeconfigs/deactivated_nodes.clj

```diff
@@ -0,0 +1,33 @@
+(ns puppetlabs.puppetdb.integration.storeconfigs.deactivated-nodes
+  (:require [clojure.test :refer :all]
+            [puppetlabs.puppetdb.integration.fixtures :as int]
+            [me.raynes.fs :as fs]))
+
+(deftest ^:integration deactivated-nodes
+  (with-open [pg (int/setup-postgres)
+              pdb (int/run-puppetdb pg {})
+              ps (int/run-puppet-server [pdb] {})]
+
+    (let [exporter-manifest  "@@notify { 'Hello from exporter': }"
+          test-collection (fn test-collection [should-be-active]
```

Comment:
`should-be-active?` or even thought about refactoring, to eliminate the boolean, i.e. vaguely:
```clojurescript
(let [...
      info #(mapv :out [(int/run-puppet-as "collector" ps pdb "Notify <<| |>>")
                        (int/run-puppet-node-status pdb "exporter")])
      verify-active #(let [[agent status] (info)]
                       (is (re-find #"Hello from exporter" agent))
                       (is (re-find #"Currently active" status)))
      verify-inactive #(let [[agent status] (info)]
                         (is (not (re-find #"Hello from exporter" agent-stdout)))
                         (is (re-find #"Deactivated at" status-stdout)))]
  ...)
```
or whatever